torch/csrc/jit/backends/backend_detail.cpp

```diff
@@ -274,6 +280,10 @@ Module codegen_backend_module(
         "] is not available. Execution of this Module is still possible by "
         "saving and loading on a device where the backend is available.");
   }
+
+  // stop debug info recording.
+  debug_info_recorder.stopRecording();
```

Comment:
Why not stop recording right after `preprocess`?

Reply:
It can be. My first implementation was RAII so I had object being destructed in mind. But if we stick with stopRecording. I will move it up.